src/controller/ordersControllers.js

```diff
@@ -1,37 +1,46 @@
 import Joi from 'joi';
 import orders from '../models/orders';
 
+export function getAllOrders (req, res) {
+	res.status(200).json({status: 'Success', Orders: orders});
```

Comment:
#### Critical
Refactor code to make it more readable. Also please follow the Airbnb style guide